code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -324,7 +324,9 @@
 /obj/item/robot_module/proc/handle_death(mob/living/silicon/robot/R, gibbed)
 	return
 
-// Medical cyborg module.
+////////////////////////////////////////
+// MARK:		Medical
+////////////////////////////////////////
```

Comment:
```suggestion
// Medical
```
In this case, if you would like to put a `MARK` for when cyborg module types actually start, that would be okay.